src/ideations/ideations.controller.ts

```diff
@@ -0,0 +1,106 @@
+import {
+    Controller,
+    Get,
+    Param,
+    Post,
+    Body,
+    Patch,
+    Delete,
+    ParseIntPipe
+} from "@nestjs/common";
+import { IdeationsService } from "./ideations.service";
+import { CreateIdeationDto } from "./dto/create-ideation.dto";
+import { UpdateIdeationDto } from "./dto/update-ideation.dto";
+import { ApiCreatedResponse, ApiTags } from "@nestjs/swagger";
+import { Ideation } from "./entities/ideation.entity";
+import { CreateIdeationVoteDto } from "./dto/create-ideation-vote.dto";
+
+@Controller()
+@ApiTags("ideations")
+export class IdeationsController {
+    constructor(private readonly ideationsService: IdeationsService) {}
+
+    //TODO updated userId to grab uuid/JWT once 
+    @Post("users/:userId/teams/:teamId/ideations")
+    @ApiCreatedResponse({type: Ideation})
+    createIdeation(
+        @Param("userId") userId: string,
+        @Param("teamId", ParseIntPipe) teamId: number,
+        @Body() createIdeationDto: CreateIdeationDto,
+    ) {
+        return this.ideationsService.createIdeation(
+            teamId, 
+            userId, 
+            createIdeationDto
+        );
+    };
+
+    @Get("/teams/:teamId/ideations")
+    @ApiCreatedResponse({type: Ideation})
+    async getIdeationsByVoyageTeam(
+        @Param("teamId", ParseIntPipe) teamId: number
+    ) {
+        const projectIdeas = await this.ideationsService.getIdeationsByVoyageTeam(teamId);
+        return projectIdeas.map((projectIdea)=>{
+            return {
+                ...projectIdea, 
+                voteCount: projectIdea.projectIdeaVotes.length
+            }
+        })
+    };
+
+    @Patch("users/:userId/ideations/:ideationId")
```

Comment:
we can pass userId in the body for now, as they will eventually come from auth/jwt

just what i've done in the other endpoint, so it's a bit more consistent when we go back and change it